### PR TITLE
Create 7_0_2_release_notes.md

### PR DESCRIPTION
**Pull request by askdba**
Creating a new file for 7.0.2 release notes.

### CODE REVIEW

**Review by deepthi — CHANGES_REQUESTED** [review]
7.0.2 release notes looking good.
Can you change the introduction for the 8.0 doc using 7.0 release notes as an example?

File: doc/releasenotes/8_0_0_release_notes.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<<<<<<< HEAD
```
Comment (deepthi) [inline doc/releasenotes/8_0_0_release_notes.md:1]: Not sure how you ended up with a merge conflict, but this needs to be resolved.

Comment (askdba) [reply doc/releasenotes/8_0_0_release_notes.md]: Should be fixed?

**Comment by deepthi** [comment]
the last two commits are missing a signoff. I think you can squash all the commits into one and as long as that has a proper signoff DCO should pass.

File: doc/releasenotes/7_0_2_release_notes.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+This update fixes several regressions that were deemed significant enough to be backported to the release branch. In addition, the experimental feature to set system variables is now behind a flag that disables it by default.
```
Comment (deepthi) [inline doc/releasenotes/7_0_2_release_notes.md:1]: Can you remove the second sentence?

**Review by deepthi — CHANGES_REQUESTED** [review]

File: doc/releasenotes/7_0_2_release_notes.md (hunk 2 above)
Comment (deepthi) [reply doc/releasenotes/7_0_2_release_notes.md:1]: Also, DCO is failing.

**Pull request closed without merging**